docs/community/chat-and-discussion-forums.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Chat & discussion forums
+description: Connect with the Filecoin community in dicussion forums or on IRC
+---
+# Chat & Discussion Forums
+
+The Filecoin community is active and here to answer your questions in your channel of choice. Please use Slack or IRC for X. For longer-lived discussions and support, our Discussion Forums are the better choice.
```

Comment:
We don't have a good policy yet on Slack vs. Discuss (and Slack is way more active) so we could strip this back for now.

```suggestion
The Filecoin community is active and here to answer your questions in your channel of choice.
```